packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -46,6 +59,25 @@ export class AtomicQuickviewModal {
     );
   }
 
+  private resizeIframe(documentWriter: Document) {
+    if (!this.iframeRef) {
+      return;
+    }
+    // This "reset" of the iframe height allows the recalculation of the proper height needed for new content being added to the iframe
+    // when the end user navigates through the available quickview
+    // Since setting a new height on the iframe will cause the resize observer to essentially "call itself", we also add a flag to stop double resizing
+    this.iframeRef.style.height = '0';
+    let shouldRecalculateResize = true;
+    new ResizeObserver(([documentElementObserved]) => {
+      if (this.iframeRef && shouldRecalculateResize) {
+        this.iframeRef.style.height = `${
+          documentElementObserved.contentRect.height + 20
```

Comment:
I prefer to avoid unnamed constants like `20` here.

Perhaps there's another property or some CSS we could use to avoid needing it?

Reply:
That just some arbitrary "breathing room", as it's uncomfortable to read text if the last line of the quickview is exactly aligned with the bottom of the iframe.

I can name it.